lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -0,0 +1,30 @@
+const Node = require('../node');
+
+
+// Returns true if the nodes are constant terms with the same constant and has an exponent each
+
+function canMultiplyLikeTermConstantNodes(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '*' || node.args[1].op === '/') {
+    return false;
+  }
+  const args = node.args;
+  if (args.some(n=> Node.PolynomialTerm.isPolynomialTerm(n))) {
```

Comment:
so I'm not sure if this is even worth asking cause it's so small, but since you're updating this file anyways can you add a space before `=>` to match the rest of em?